docs/user/analytics.rst

```diff
@@ -19,7 +19,7 @@ and then click on :guilabel:`Traffic Analytics`.
 
    Traffic analytics demo
 
-You can also access to analytics data from :ref:`search results <server-side-search/index:Search Analytics>`.
```

Comment:
I kinda like the refs because they hoverxref. Should we standardize on using them instead of `:doc:`?

Reply:
> I kinda like the refs because they hoverxref.

This is true. I would say that we should add actual labels to headlines so we can freely change headlines and not have to update references as a result (that adds more work to a simple headline improvement).

Reply:
I think we should defer to https://github.com/readthedocs/sphinx-hoverxref/issues/18 because of the widespread use and need of `:doc:`. But would be great to fix this.